styles.css

```diff
@@ -0,0 +1,120 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto+Serif:opsz,wght@8..144,100&display=swap');
+
+@media screen and (max-width: 640px) {
+  main {
+    padding: 0;
+    margin: 0;
+  }
+
+  header,
+  section {
+    margin: 0;
+    padding: 20px;
+    color: #fff;
+  }
+
+  header {
+    background: #3e3c3c;
+    display: flex;
+    justify-content: space-between;
+  }
```

Comment:
- [ ] Kindly make sure to align the `header` content correctly. All content should be aligned in the center. Kindly use `alig-items: center` and everything will be aligned correctly. :wink: